code/core/src/highlight/useHighlights.stories.tsx

```diff
@@ -1,16 +1,55 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 
-import { mockChannel } from 'storybook/internal/preview-api';
+import { useChannel } from 'storybook/internal/preview-api';
```

Comment:
Very impressive how you got this in a story, and truly can show of the functionality working.

The highlighting, uses the real highlighting code, via the real addon, but then the click handlers used addon actions, that did confuse me. I'd expect the click functionality to also be "real" then.

Reply:
That would assume addon-a11y is around, which I don't love. Right now it asserts all of the functional surface area, including the fact that this clickEvent is emitted and can be listened to on the (real) channel.

I'll update the event name but leave the rest as-is.